src/utils/analysis-utils.ts

```diff
@@ -434,8 +436,25 @@ export async function loadFeaturesClientSide(
 }
 
 export function getAnalysisTableColumns(
-  analysisResult: BaselineLayerResult,
+  analysisResult: BaselineLayerResult | PolygonAnalysisResult,
 ): Column[] {
+  if ('tableColumns' in analysisResult) {
+    return analysisResult.tableColumns.map(column => {
+      return {
+        id: column as any,
+        // remove prefix from column labels
+        // example: replace "stat:area" with "area"
+        label: column.replace(/^[a-z]+:/i, ''),
+        format: (value: number | string) => {
+          if (typeof value === 'number') {
+            return value.toLocaleString('en-US');
+          }
+          return isNil(value) ? 'null' : value;
```

Comment:
putting null in string here also feels dangerous. If this is something we're showing to the user, I'd prefer something like `No value` since to a computer parsing `'null'` could have unintended side-effects esp in JS.

Reply:
fixed.  metadata properties that start with "class:" now say "no class".  Data values now use getRoundedData, so show "no data" translated.